packages/data-client-core/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@terra-ui-packages/data-client-core",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "vite build --emptyOutDir",
+    "dev": "vite build --mode=development --watch",
+    "test": "jest"
+  },
+  "type": "module",
+  "module": "./lib/es/index.js",
+  "main": "./lib/cjs/index.cjs",
+  "types": "./lib/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.cjs",
+      "types": "./lib/types/index.d.ts"
+    }
+  },
+  "files": [
+    "lib/cjs/**",
+    "lib/es/**",
+    "lib/types/**/*.d.ts"
+  ],
+  "devDependencies": {
+    "@terra-ui-packages/build-utils": "^1.2.0",
+    "@terra-ui-packages/test-utils": "*",
```

Suggest using the latest `build-utils` and a version specifier for `test-utils`.

```suggestion
    "@terra-ui-packages/build-utils": "^2.0.0",
    "@terra-ui-packages/test-utils": "^1.0.0",
```

Same in the other package.